components/reddit/common.js

```diff
@@ -0,0 +1,32 @@
+const reddit = require("./reddit.app");
```

Comment:
`common.js` should live under `sources/` since it's relative to event sources specifically. As soon as we start adding actions, we'll have the equivalent `actions/common.js` file for that.

Reply:
ok, i have move the file under `sources/`